save-frontend-common/src/main/kotlin/com/saveourtool/save/frontend/common/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -0,0 +1,460 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
```

Comment:
why it's moved to the common module?

Reply:
@Cheshiriks , please speed up and fix these problems

Reply:
Moved it to cosv-frontend